tests/admin_inlines/tests.py

```diff
@@ -86,6 +87,23 @@ def test_inline_primary(self):
         self.assertEqual(response.status_code, 302)
         self.assertEqual(len(Fashionista.objects.filter(person__firstname='Imelda')), 1)
 
+    def test_inline_primary_with_uuids(self):
```

Comment:
At this point I can't even remember which of my "wrong" fixes broke this test, and I agree that it doesn't gain anything over the current version, so I'll just remove it.